editor/editor_property_name_processor.cpp

```diff
@@ -318,6 +338,26 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 			"then",
 			"to",
 	});
+
+	// Translation context associated with a name.
+	// The second key is either:
+	// - `full/property/path`
+	// - `Class::full/property/path`
+	// In case a class name is needed to distinguish between usages, all usages should use the second format.
+	//
+	// The following initialization is parsed in `editor/translations/scripts/common.py` with a regex.
+	// The map name and value definition format should be kept synced with the regex.
+	translation_contexts["force"]["constant_force"] = "Physics";
+	translation_contexts["force"]["force/8_bit"] = "Enforce";
+	translation_contexts["force"]["force/mono"] = "Enforce";
+	translation_contexts["force"]["force/max_rate"] = "Enforce";
+	translation_contexts["force"]["force/max_rate_hz"] = "Enforce";
+	translation_contexts["normal"]["theme_overrides_font/normal"] = "Ordinary";
```

Comment:
Which class uses this?

Reply:
There's a "normal" font in default theme:

https://github.com/godotengine/godot/blob/dce1aab174369e9124507f4614fc007301df52e7/scene/theme/default_theme.cpp#L934-L934

Extractor adds `theme_overrides_TYPE/` prefix to all detected theme item names.

This is never displayed since `Fonts` type is not used directly in `Control` I think. It's here to prevent generating a "normal" entry without context for that line of code.

Reply:
Looks like this entry is unused. It's here since Godot was open-sourced, but not referenced anymore.
You can delete it and the associated context.